python_modules/dagster/dagster/_core/remote_representation/origin.py

```diff
@@ -274,9 +274,11 @@ def create_single_location(
         from dagster._core.remote_representation.code_location import GrpcServerCodeLocation
         from dagster._core.remote_representation.grpc_server_registry import GrpcServerRegistry
         from dagster._core.workspace.context import WEBSERVER_GRPC_SERVER_HEARTBEAT_TTL
+        from dagster._grpc.server import GrpcServerCommand
 
         with GrpcServerRegistry(
             instance_ref=instance.get_ref(),
+            server_command=GrpcServerCommand.API_GRPC,
```

Comment:
to the best of my knowledge this method is only used in tests. any reason not to make it use the same path that most everything else will be using soon?

Reply:
I just lacked context so left it alone. I'm happy to change it.